backend/client.go

```diff
@@ -408,8 +415,14 @@ func (c *client) SendAnswer(ctx context.Context, pl Answer) error {
 		return errors.Wrap(err, "json marshal error")
 	}
 
-	// TODO add context for cancellation
-	resp, err := c.httpClient.Post(c.server, "application/json", bytes.NewReader(b))
+	req, err := http.NewRequestWithContext(ctx, http.MethodPost, c.server, bytes.NewReader(b))
+	if err != nil {
+		return errors.Wrap(err, "new request error")
+	}
+	req.Header.Add("Content-Type", "application/json")
+	req.Header.Add("Authorization", c.authorization)
```

Comment:
Maybe we should wrap this within a `if c.authorization != ""` to make sure that the `Authorization` header is not set when this option is not configured?